politeiawww/middleware.go

```diff
@@ -137,3 +139,57 @@ func recoverMiddleware(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r)
 	})
 }
+
+func (p *politeiawww) getPassThrough(route string, r *http.Request) ([]byte, error) {
+	dest := cms.ProposalsMainnet
+	if p.cfg.TestNet {
+		dest = cms.ProposalsTestnet
+	}
+	route = dest + route
+
+	resp, err := http.Get(route)
+	if err != nil {
+	}
+	defer resp.Body.Close()
+
+	return ioutil.ReadAll(resp.Body)
+}
+
+func (p *politeiawww) postPassThrough(route string, r *http.Request) ([]byte, error) {
+	dest := cms.ProposalsMainnet
+	if p.cfg.TestNet {
+		dest = cms.ProposalsTestnet
+	}
+
+	versionRoute := dest + "/api/v1" + www.RouteVersion
+
+	resp, err := http.Get(versionRoute)
```

Comment:
Why are you passing the request in here?  The calling function should not have to make a request to use this function.  That should be handled in the function itself.  If all you're using is the request body from it then just pass the request body in so its not confusing.